.github/workflows/integration-tests-api.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration test - API
+on: pull_request
+
+jobs:
+  runTests:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      checks: write
+    strategy:
+      matrix:
+        node-version: ['lts/*'] # 18.17.1 or lts/* 
+        test-pattern:
+          - accounts.test.ts
+          - addresses.test.ts
+          - batches.test.ts
+          - blocks.test.ts
+          - contracts.test.ts
+          - logs.test.ts
+          - stats.test.ts
+          - tokens.test.ts
+          - transactions.test.ts
+    name: 'API test set: ${{ matrix.test-pattern}} / Node: ${{ matrix.node-version}}'
+    steps:
+      - name: Checkout with Submodule
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: npm
+
+      - name: Install dependencies
+        run: |
+          npm ci --no-audit
+          npx playwright install --with-deps chromium
```

Comment:
Why is `playwright install` here? Should we move it to the installation script, maybe postinstall? Otherwise locally I'll have to do the same.

Reply:
updated